front-end/src/lib/store.js

```diff
@@ -1,7 +1,8 @@
 import { createStore, applyMiddleware } from 'redux'
 import Thunk from 'redux-thunk'
```

Comment:
@Jeffzholy imported functions and variables should start with lower char, only components and classes start with capital